<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NumPadLogInDialog</class>
 <widget class="QDialog" name="NumPadLogInDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NumPadLogInDialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QLCDNumber" name="lcdNumber"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="DuNumericKeyPushButton" name="sietePushButton">
         <property name="text">
          <string>7</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="DuNumericKeyPushButton" name="cuatroPushButton">
         <property name="text">
          <string>4</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="DuNumericKeyPushButton" name="unoPushButton">
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearPushButton">
         <property name="text">
          <string>CLEAR</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="DuNumericKeyPushButton" name="ochoPushButton">
         <property name="text">
          <string>8</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="DuNumericKeyPushButton" name="cincoPushButton">
         <property name="text">
          <string>5</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="DuNumericKeyPushButton" name="dosPushButton">
         <property name="text">
          <string>2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="DuNumericKeyPushButton" name="ceroPushButton">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="DuNumericKeyPushButton" name="nuevePushButton">
         <property name="text">
          <string>9</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="DuNumericKeyPushButton" name="seisPushButton">
         <property name="text">
          <string>6</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="DuNumericKeyPushButton" name="tresPushButton">
         <property name="text">
          <string>3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="enterPushButton">
         <property name="text">
          <string>ENTER</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DuNumericKeyPushButton</class>
   <extends>QPushButton</extends>
   <header location="global">dunumerickeypushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
